src/Hive.Core/sia380/Hive.Core.sia380.json

```diff
@@ -157,7 +157,7 @@
       "subcategory": "Demand",
       "id": "8903e11e-abba-4318-a9b2-6728e2999652",
       "parameter-type": "ValueList",
-      "csv": "200728_SIA2024_Raumdaten_Standardwert.csv"
+      "csv": "201008_SIA2024_Raumdaten_Standardwert.csv"
```

Comment:
@daren-thomas I wanted to make this a relative path, like in the `compile_sia2024_json.py´:
```
file_template = "201008_SIA2024_Raumdaten_Standardwert.csv"
folder = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "Resources")) 
file_path = os.path.join(folder, file_template)
```
but because this is a json, I didn't know how. Do you have any idea? Currently, `201008_SIA2024_Raumdaten_Standardwert.csv´ is just copy pasted from `Hive.IO\Resources` into `Hive.Core\sia380`, but of course it would be better to not have databases double.

Same for lines 171 and 182 in this json

Reply:
you _could_ use `..` to denote a parent directory. depending on who is reading this and how. ah wait. it's in the badgerfile. So yes. `..\..\..\path\to\csv\my_csv_file.csv` should work. want to give it a shot?

Reply:
![image](https://user-images.githubusercontent.com/21927657/98695928-7c62e800-2373-11eb-86bb-787e44fda83a.png)

huhh... what happened...? The list components work with the `..\\..`, but it doesn't load the other components anymore?

![image](https://user-images.githubusercontent.com/21927657/98696049-9dc3d400-2373-11eb-8b60-7a33db4f9b13.png)

Reply:
building the badger files seems alright (see below)... Main branch also still works. But trying this branch always gives me the error above when launching Grasshopper

```
C:\Users\Christoph\Documents\GitHub\hive\src\Hive.Core>REM use honey-badger to build all the Hive.Core components and install them to Libraries

C:\Users\Christoph\Documents\GitHub\hive\src\Hive.Core>REM set up some environment variables to make this easier

C:\Users\Christoph\Documents\GitHub\hive\src\Hive.Core>REM (this assumes the IronPython installation folder and also the path to honey-badger.py)

C:\Users\Christoph\Documents\GitHub\hive\src\Hive.Core>SET IPY=C:\Program Files\IronPython 2.7\ipy.exe

C:\Users\Christoph\Documents\GitHub\hive\src\Hive.Core>SET HB=C:\Users\Christoph\Documents\GitHub\honey-badger\honey-badger.py

C:\Users\Christoph\Documents\GitHub\hive\src\Hive.Core>"C:\Program Files\IronPython 2.7\ipy.exe" "C:\Users\Christoph\Documents\GitHub\honey-badger\honey-badger.py" -i epw_reader\Hive.Core.epw_reader.json
installing Hive.Core.epw_reader.ghpy to C:\Users\Christoph\AppData\Roaming\Grasshopper\Libraries
done.

C:\Users\Christoph\Documents\GitHub\hive\src\Hive.Core>"C:\Program Files\IronPython 2.7\ipy.exe" "C:\Users\Christoph\Documents\GitHub\honey-badger\honey-badger.py" -i sia380\Hive.Core.sia380.json
installing Hive.Core.sia380.ghpy to C:\Users\Christoph\AppData\Roaming\Grasshopper\Libraries
installing Hive.Core.sia380.gha to C:\Users\Christoph\AppData\Roaming\Grasshopper\Libraries
installing honey-badger-runtime.dll to C:\Users\Christoph\AppData\Roaming\Grasshopper\Libraries
done.

C:\Users\Christoph\Documents\GitHub\hive\src\Hive.Core>"C:\Program Files\IronPython 2.7\ipy.exe" "C:\Users\Christoph\Documents\GitHub\honey-badger\honey-badger.py" -i solar\Hive.Core.solar.json
installing Hive.Core.solar.ghpy to C:\Users\Christoph\AppData\Roaming\Grasshopper\Libraries
done.

C:\Users\Christoph\Documents\GitHub\hive\src\Hive.Core>"C:\Program Files\IronPython 2.7\ipy.exe" "C:\Users\Christoph\Documents\GitHub\honey-badger\honey-badger.py" -i solar_tech\Hive.Core.solar_tech.json
installing Hive.Core.solar_tech.ghpy to C:\Users\Christoph\AppData\Roaming\Grasshopper\Libraries
installing Hive.Core.solar_tech.gha to C:\Users\Christoph\AppData\Roaming\Grasshopper\Libraries
installing honey-badger-runtime.dll to C:\Users\Christoph\AppData\Roaming\Grasshopper\Libraries
done.

C:\Users\Christoph\Documents\GitHub\hive\src\Hive.Core>"C:\Program Files\IronPython 2.7\ipy.exe" "C:\Users\Christoph\Documents\GitHub\honey-badger\honey-badger.py" -i combustion\Hive.Core.combustion.json
installing Hive.Core.combustion.ghpy to C:\Users\Christoph\AppData\Roaming\Grasshopper\Libraries
done.

C:\Users\Christoph\Documents\GitHub\hive\src\Hive.Core>"C:\Program Files\IronPython 2.7\ipy.exe" "C:\Users\Christoph\Documents\GitHub\honey-badger\honey-badger.py" -i cooling\Hive.Core.cooling.json
installing Hive.Core.cooling.ghpy to C:\Users\Christoph\AppData\Roaming\Grasshopper\Libraries
done.

C:\Users\Christoph\Documents\GitHub\hive\src\Hive.Core>"C:\Program Files\IronPython 2.7\ipy.exe" "C:\Users\Christoph\Documents\GitHub\honey-badger\honey-badger.py" -i heatpumps\Hive.Core.heatpumps.json
installing Hive.Core.heatpumps.ghpy to C:\Users\Christoph\AppData\Roaming\Grasshopper\Libraries
done.

C:\Users\Christoph\Documents\GitHub\hive\src\Hive.Core>pause
Drücken Sie eine beliebige Taste . . .
```

Reply:
it looks like you need to escape backslashes in JSON. I'll take a look.

Reply:
![image](https://user-images.githubusercontent.com/2969564/98784174-02764180-23fb-11eb-9d88-797e96ff8f80.png)

I'm not sure I understand what you mean? seems to be working for me...

Reply:
This was fixed by updating honey-badger and recompiling the honey-badger-runtime solution.